openroad.bzl

```diff
@@ -828,20 +817,15 @@ def build_openroad(
     )
 
     if mock_area != None:
-        mock_area_stages(target_name, name, stage_sources, io_constraints, sdc_constraints, stage_args, outs, variant, mock_area, docker_image, external_pdk)
+        mock_area_stages(target_name, name, stage_sources, stage_args, outs, variant, mock_area, docker_image, external_pdk)
 
     # _make targets
     for (previous, stage) in zip(["n/a"] + stages, stages):
         # Generate config for stage targets
-        stage_cfg_srcs = []
-        if sdc_constraints != None:
-            stage_cfg_srcs.append(sdc_constraints)
-        if io_constraints != None:
-            stage_cfg_srcs.append(io_constraints)
         write_stage_config(
             name = target_name + "_" + stage + "_config",
             stage = stage,
-            srcs = stage_cfg_srcs,
+            srcs = stage_sources[stage],
```

Comment:
Adding all stage's sources to `srcs` of config targets, changes how `_scripts` target behave. Apart from, creating scripts it will try to generate dependencies which are not required for configs.

This is also the reason why the CI fails, as it tries to build additional dependencies with the Docker flow, where only the local flow should be tested